CHANGELOG.md

```diff
@@ -1,49 +1,42 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+The Changelog file gives an overview of all of the notable changes affecting the project.
+The format of it is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/), and the project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+For public Changelog covering all changes done to Pipedrive’s API, webhooks and app extensions platforms, see [public Changelog](https://pipedrive.readme.io/docs/changelog) with discussion area in [Developers Community](https://devcommunity.pipedrive.com/c/documentation/changelog/19).
 
 ## [Unreleased]
 ### Changed
 - Changed POST /v1/webhooks to accept `application/json` instead of `application/x-www-form-urlencoded` to reflect the reality
 ### Added
-- added required flag for pipeline name in `POST /pipelines` endpoint documentation
+* Support for endpoints added and changed endpoints changed after March 31, 2021
+* Support for multiple instances [issue #176](https://github.com/pipedrive/client-nodejs/issues/176)
+* Convenience functions regarding OAuth 2.0 lifecycle including `buildAuthorizationUrl`, `authorize` and `refreshToken`
+* Support for changing the host URL [issue #228](https://github.com/pipedrive/client-nodejs/issues/228)
+* Support for both snake_case and camelCase
+* `UpdateProductResponse` for `PUT /products/{id}`
+* `Team` (singular) schema for endpoints that deal with a single team
+* Added support for custom fields for `Content-Type: application/x-www-form-urlencoded`
+* Add into documentation that also a Lead is one of the possible returned entities in related objects for the search endpoint.
```

Comment:
Should it be in the **Changed** section? something like:
- Updated documentation for search endpoint: include Lead as one of the possible returned entities in related objects